app/access_policies/user_access_policy.rb

```diff
@@ -9,7 +9,7 @@ def self.action_allowed?(action, requestor, user)
     when :search
       requestor.is_application? || requestor.is_activated?
     when :read, :update
-      requestor.is_human? && requestor.is_activated? && \
+      requestor.is_human? && \
```

Comment:
Makes it so users who haven't completed registration can update their profile (change their email addresses)